test/isIndexed.js

```diff
@@ -0,0 +1,28 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('isIndexed', function() {
+  context('given a `String` value', function() {
+    specify('should return true', function() {
+      assert.isTrue(RA.isIndexed('test'));
+      assert.isTrue(RA.isIndexed(Object('test')));
```

Comment:
Let's add empty string check
```suggestion
      assert.isTrue(RA.isIndexed(Object('test')));
      assert.isTrue(RA.isIndexed(''));
```